Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -368,6 +486,28 @@ public class CertificateValidationRequest : CommonRequest {
     }
 }
 
+/// This is a simplified adaptation of the _getDataToHash function from
+/// https://github.com/digitalbazaar/jsonld-signatures. 
+/// This function prefixes the normalized json-ld with optional headers. Blockcerts only uses the 
+/// created date header for now.
+private func _getDataToHash(input: String, date: String) -> String {
+    let toHash = "http://purl.org/dc/elements/1.1/created: " + date + "\n" + input
+    return toHash
+}
+
+
+private  func hexStringFromData(input: NSData) -> String {
```

Comment:
https://github.com/blockchain-certificates/cert-wallet/blob/master/Frameworks/BlockchainCertificates/Extensions/String.swift#L49-L58

Admittedly, that file should be renamed, since it's not **just** for string extensions.